### PR TITLE
add setting for which countries to get phone numbers from

### LINKED ISSUE #501: Add a setting to customize countries which we accept phones for
Add a setting to allow us to customize which country codes to collect phone numbers for, rather than hardcoding it in the [helper function](https://github.com/DoSomething/voting-app/blob/dev/app/Http/helpers.php#L34-L42).

### PR DESCRIPTION
**Pull request by katiecrane**
- admins can add country codes to list, separated by commas with no spaces
- database migration to add new setting
### Issues:

Fixes #501

### CODE REVIEW

File: app/Http/helpers.php [hunk 1]
```diff
@@ -49,7 +49,9 @@ function dosomething_global_countries()
  */
 function should_collect_phone()
 {
-    return in_array(get_country_code(), dosomething_global_countries());
```
Comment (DFurnes) [inline app/Http/helpers.php:52]: Was this the only place we used `dosomething_global_countries()`? If so, we should remove it.

Comment (katiecrane) [inline app/Http/helpers.php:52]: I'll double check

Comment (katiecrane) [inline app/Http/helpers.php:52]: It was! It's dead now RIP

**Comment by DFurnes** [comment]
:+1:

**Pull request merged**